src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let amount = 0;
+
+  if (days < 3) {
+    amount = (40 * days);
+  }
+
+  if ((days >= 3) && (days < 7)) {
+    amount = (40 * days) - 20;
+  }
+
+  if (days >= 7) {
+    amount = (40 * days) - 50;
+  }
+
+  return amount;
```

Comment:
Create variables for the values so that you can reuse them and give them descriptive names.
Avoid reassigning variables values. It's better to use `const` instead of `let`
You should return the value immediately in if statements. And don't use `else` after `return `statement.

Reply:
If I understood you correctly, it should be correct now